app/src/main/java/com/SecUpwN/AIMSICD/adapters/AIMSICDDbAdapter.java

```diff
@@ -2161,9 +2167,18 @@ public void toEventLog(int DF_id, String DF_desc){
 
                 mDb.insert("EventLog", null, eventLog);
                 Log.i(TAG, "ToEventLog(): Added new event: id=" + DF_id + " time=" + time + " cid=" + cid);
+
                 // Short 100 ms Vibration
-                Vibrator v = (Vibrator) mContext.getSystemService(Context.VIBRATOR_SERVICE);
-                v.vibrate(100);
+                // TODO not elegant solution, vibrator invocation should be moved somewhere else imho
+                Log.d(TAG, String.format("vibrate enabled:%b",mContext.getString(R.string.pref_notification_vibrate_enable)));
```

Comment:
I'm sorry, I shouldn't have left the Log call in there, it was just for debugging purpose, will remove it.